DEVELOPMENT.md

```diff
@@ -0,0 +1,496 @@
+# KCP-Edge Contributor Development Environment
+
+## Required Packages:
+   - ko: https://ko.build/install/ 
+   - gcc
+   - docker 
+   - jq
+   - make
+   - go (version expected 1.19)
+   - kind
+   - kubectl (version range expected: 1.23-1.25)
+
+
+## Supported OS Platforms 
+  - Linux
+  - MacOS
+  - Windows WSL/Ubuntu
+
+
+After cloning this repo (see quickstart), run the following script to install the required package (Linux or MacOS):
```

Comment:
I suppose the intended referent is the "### 1. Clone the kcp-edge repo" section below; the existing reference to "quickstart" does not obviously lead there.